src/Microsoft.AspNetCore.Razor.Evolution/RazorCompilation.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Razor.Evolution
+{
+    public class RazorCompilation
```

Comment:
This should have a `RazorProject` not a `RazorProjectItem`. The item/path can be a parameter 👍